easybuild/easyconfigs/u/UDUNITS/UDUNITS-2.2.25-gmvolf-17.02.eb

```diff
@@ -0,0 +1,119 @@
+# Built with EasyBuild version 2.3.0 on 2015-10-29_16-55-56
```

Comment:
Could you please remove the stale comments (build stats below and first three lines)?